Epam.TestAutomation.Tests.BDD/Features/Epam_DDT_SelectedLocations.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Epam_DDT_SelectedLocations
+As an automation engineer
+I want to test Epam site for career opportunities
+by selecting different locations
+and verify that returned results match
+
+    @automation @DDT
+    Scenario Outline: Check that search results contain selected locations
```

Comment:
Please review scenario naming. It better to name it like Module – Submodule – Test. Ex: Job Listing - Filtering Panel - Check vacancies with relocation.